force-app/main/default/classes/EmailServiceInstaller.cls

```diff
@@ -88,4 +88,66 @@ public with sharing class EmailServiceInstaller {
         Http h = new Http();
         HttpResponse res = h.send(req);
     }
+
+    public static void deleteEmailServiceFunction(EmailServicesFunction emailServiceFunction)
```

Comment:
I would add a unit test for this if possible

Reply:
The PR comment literally addresses this exact thing.